admin/tools/docker/dev/Dockerfile

```diff
@@ -2,10 +2,11 @@
 FROM qserv/qserv:dev
 MAINTAINER Fabrice Jammes <fabrice.jammes@in2p3.fr>
 
-USER qserv 
+USER root
```

Comment:
Default value seems to be undocumented, can't it be the same than in the parent image?